Code/Tools/FBuild/FBuildCore/Graph/DLLNode.cpp

```diff
@@ -37,6 +37,21 @@ DLLNode::~DLLNode()
 {
 }
 
+// DoBuild
+//------------------------------------------------------------------------------
+/*virtual*/ Node::BuildResult DLLNode::DoBuild( Job * job )
+{
+	// Make sure the implib output directory exists
+	if ( m_ImportLibName.IsEmpty() == false)
+	{
+		AStackString<> cleanPath;
+		NodeGraph::CleanPath( m_ImportLibName, cleanPath );
+		EnsurePathExistsForFile( cleanPath );
```

Comment:
Need to handle an error here:
	if ( EnsurePathExistsForFile( cleanPath ) == false )
	{
		return NODE_RESULT_FAILED; // EnsurePathExistsForFile will have emitted error
	}